client/blocks/qr-code-login/index.jsx

```diff
@@ -185,11 +185,17 @@ function QRCodeLogin( { locale, redirectToAfterLoginUrl } ) {
 	// Send the user to the login state.
 	useEffect( () => {
 		if ( authState?.auth_url ) {
+			// if redirect URL is set, append to to the response URL as a query param.
+			if ( redirectToAfterLoginUrl ) {
+				authState.auth_url = addQueryArgs( authState?.auth_url, {
```

Comment:
We know this is defined.

```suggestion
				authState.auth_url = addQueryArgs( authState.auth_url, {
```